src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/use_layout_styles.tsx

```diff
@@ -55,7 +58,7 @@ export const useLayoutStyles = () => {
       }
 
       .kbnGridPanel--resizeHandle {
-        z-index: ${euiTheme.levels.mask};
+        z-index: ${euiTheme.levels.maskBelowHeader};
```

Comment:
This makes it so that the drag handle no longer overlaps the sticky bar in Dashboard

| Before | After |
|--------|--------|
| ![Screenshot 2025-01-23 at 5 49 13 PM](https://github.com/user-attachments/assets/ea5f2ffe-46bd-4259-920a-8798a1a08775) | ![image](https://github.com/user-attachments/assets/52a6ac63-5bb3-4cc0-a77d-99cac756aaa4) |